src/pages/ThorChainLP/hooks/useUserLpData.ts

```diff
@@ -0,0 +1,108 @@
+  totalPoolUnits: string
+}): string => bn(userLiquidityUnits).div(totalPoolUnits).times(100).toFixed()
+
+export const thorchainLendingPositionQueryFn = async ({
+  queryKey,
+}: {
+  queryKey: [string, { accountId: AccountId; assetId: AssetId }]
+}) => {
+  const [, { accountId, assetId }] = queryKey
+  const position = await getThorchainLendingPosition({ accountId, assetId })
+  return position
+}
+
+export const useUserLpData = ({ accountId, assetId }: UseLendingPositionDataProps) => {
+  // TODO(gomes): return a list of positions, and discriminate asym/sim as two different positions
+  // TODO(gomes): handle symmetric LP positions - this work was started with an asymmetric position with underlying rebalancing
+  const lpPositionQueryKey: [string, { accountId: AccountId | undefined; assetId: AssetId }] =
+    useMemo(() => ['thorchainUserLpData', { accountId, assetId }], [accountId, assetId])
+
+  const poolAssetMarketData = useAppSelector(state => selectMarketDataById(state, assetId))
+  const runeMarketData = useAppSelector(state => selectMarketDataById(state, thorchainAssetId))
+
+  const lendingPositionData = useQuery({
+    // TODO(gomes): remove me, this avoids spamming the API during development
+    staleTime: Infinity,
+    queryKey: lpPositionQueryKey,
+    queryFn: async ({ queryKey }) => {
+      const [, { accountId, assetId }] = queryKey
+
+      if (!accountId) return null
+      const position = await getThorchainLiquidityProviderPosition({ accountId, assetId })
+
+      if (!position) return null
+
+      const poolAssetId = assetIdToPoolAssetId({ assetId })
+      const { REACT_APP_THORCHAIN_NODE_URL } = getConfig()
+
+      const { data: poolData } = await axios.get<ThornodePoolResponse>(
+        `${REACT_APP_THORCHAIN_NODE_URL}/lcd/thorchain/pool/${poolAssetId}`,
+      )
+      return { position, poolData }
+    },
+    select: data => {
+      if (!data) return null
+
+      const { position, poolData } = data
+
+      const underlyingAssetValueFiatUserCurrency = fromThorBaseUnit(
+        position?.asset_deposit_value || '0',
+      ).times(poolAssetMarketData?.price || 0)
+      const underlyingRuneValueFiatUserCurrency = fromThorBaseUnit(
+        position?.rune_deposit_value || '0',
+      ).times(runeMarketData?.price || 0)
+
+      const isAsymmetric =
+        bnOrZero(position.assetAdded).isZero() || bnOrZero(position.runeAdded).isZero()
+      const asymSide = bn(position.assetDeposit).gt(position.asset_deposit_value) ? 'asset' : 'rune'
+
+      const totalValueFiatUserCurrency = underlyingAssetValueFiatUserCurrency
+        .plus(underlyingRuneValueFiatUserCurrency)
+        .toFixed()
+
+      const poolOwnershipPercentage = calculatePoolOwnershipPercentage({
+        userLiquidityUnits: position.liquidityUnits,
+        totalPoolUnits: poolData.pool_units,
+      })
+
+      return {
+        underlyingAssetAmountCryptoPrecision: fromThorBaseUnit(
+          position.asset_deposit_value,
```

Comment:
Note, THOR only returns us the value at the time of deposit, i.e when depositing 0.01 ETH, 0.005 ETH and matching 0.23506872 RUNE.
To get the rebalanced value, we will need to calculate manually, see THORYield

<img width="435" alt="image" src="https://github.com/shapeshift/web/assets/17035424/c78eebd0-1cdc-4e09-b304-6671366614d3">

Notice how there's now more underlying RUNE and less underlying ETH.